drivers/health_pctp/app/models/health_pctp/careplan.rb

```diff
@@ -0,0 +1,266 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HealthPctp
+  class Careplan < HealthBase
+    include Rails.application.routes.url_helpers
+
+    phi_patient :patient_id
+    phi_attr :user_id, Phi::SmallPopulation
+    phi_attr :completed_at, Phi::Date
+
+    belongs_to :patient, class_name: 'Health::Patient', optional: true
+    belongs_to :user, optional: true
+
+    has_many :needs, dependent: :destroy
+    has_many :care_goals, dependent: :destroy
+
+    scope :in_progress, -> { where(patient_signed_on: nil) }
+    scope :completed_within, ->(range) { where(patient_signed_on: range) }
+
+    scope :rn_approved, -> { where.not(reviewed_by_rn_on: nil) }
+    scope :reviewed_within, ->(range) { where(reviewed_by_rn_on: range) }
+
+    scope :allowed_for_engagement, -> do
+      joins(patient: :patient_referrals).
+        merge(
+          ::Health::PatientReferral.contributing.
+            where(
+              hpr_t[:enrollment_start_date].lt(Arel.sql("#{arel_table[:patient_signed_on].to_sql} + INTERVAL '1 year'")),
+            ),
+        )
+    end
+
+    scope :editable, -> { where(patient_signed_on: nil) }
+
+    alias_attribute :completed_at, :patient_signed_on
+    alias_attribute :initial_date, :created_at
+    alias_attribute :careplan_sent_on, :sent_to_pcp_on
+
+    def active?
+      completed? && patient_signed_on >= 1.years.ago
+    end
+
+    def editable?
+      patient_signed_on.nil?
+    end
+
+    def completed?
+      patient_signed_on.present?
+    end
+
+    def cp1?
+      false
+    end
+
+    def cp2?
+      true
+    end
+
+    def edit_path
+      edit_client_health_pctp_careplan_path(patient.client, id)
+    end
+
+    def show_path
+      client_health_pctp_careplan_path(patient.client, id)
+    end
+
+    def identifying_information
+      {
+        name: ['Enrollee Name', :string, nil],
+        dob: ['DOB', :date_picker, nil],
+        phone: ['Enrollee Phone Number', :tel, nil],
+        email: ['Enrollee Email Address', :email, nil],
+        mmis: ['MMIS', :string, nil],
+        aco: ['ACO/MCO', :string, nil],
+      }
+    end
+
+    def care_team_members
+      {
+        cc: "Enrollee's Care Coordinator (s)",
```

Comment:
To be consistent with the line below.
```suggestion
        cc: "Enrollee's Care Coordinator(s)",
```